index.js

```diff
@@ -46,12 +51,18 @@ class Conf {
 			...options
 		};
 
+		const getPackageData = memoizeOne(() => {
```

Comment:
You don't really need memoization here, you can just use https://github.com/sindresorhus/onetime

Reply:
👍